Assets/Plugins/Android/Editor/AndroidBuilder.cs

```diff
@@ -265,7 +266,7 @@ private static string GetBuildTools()
 
         private static void WriteConfigMacros(string filepath)
         {
-            var contents = File.ReadAllText(filepath);
+            var contents = FileUtility.ReadAllText(filepath);
```

Comment:
Just changes where the file reading is happening to a central place, to be able to do platform-dependent things with the implementation.